### PR TITLE
Mod/8529 add tooltip and glossary to time period filter

### PR DESCRIPTION
**Pull request by brianpetway**
**High level description:**

Add a tooltip to the 'Time Period' filter on the Advanced Search page. Also add a glossary link for 'fiscal year' after the Fiscal Year tab in the Time Period filter.

**Technical details:**

Followed the established pattern to add the tooltip. Note that Jessica decided to go with bold text for the glossary terms in the tooltip rather than have links in there, for accessibility reasons.

**JIRA Ticket:**
[DEV-8529](https://federal-spending-transparency.atlassian.net/browse/DEV-8529)

**Mockup:**
n/a

The following are ALL required for the PR to be merged:

Author:
- [x ] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x ] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x ] Verified mobile/tablet/desktop/monitor responsiveness
- [x ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/js/components/search/filters/timePeriod/TimePeriod.jsx [hunk 1]
```diff
@@ -310,7 +311,7 @@ export default class TimePeriod extends React.Component {
                                 aria-label="Fiscal Year"
                                 title="Fiscal Year"
                                 onClick={this.toggleFilters}>
-                                Fiscal Year
+                                Fiscal Year<GlossaryLink term="fiscal-year-fy" />
```
Comment (nick-torres) [inline src/js/components/search/filters/timePeriod/TimePeriod.jsx:314]: do you need to style this glossary icon so that it's gray not blue?

Comment (brianpetway) [reply src/js/components/search/filters/timePeriod/TimePeriod.jsx:314]: good catch.
fixed.

File: src/js/components/glossary/Glossary.jsx [hunk 2]
```diff
@@ -22,6 +22,7 @@ const propTypes = {
 
 export default class Glossary extends React.Component {
     constructor(props) {
+        console.log('glossary cmpnt');
```
Comment (nick-torres) [inline src/js/components/glossary/Glossary.jsx:25]: console.logssss

File: src/js/components/glossary/Glossary.jsx [hunk 3]
```diff
@@ -96,6 +97,7 @@ export default class Glossary extends React.Component {
     }
 
     render() {
+        console.log('glossary render');
```
Comment (nick-torres) [inline src/js/components/glossary/Glossary.jsx:100]: console.logggsss

File: src/js/components/glossary/Glossary.jsx (hunk 2 above)
Comment (brianpetway) [reply src/js/components/glossary/Glossary.jsx]: remooooooved

File: src/js/components/glossary/Glossary.jsx (hunk 3 above)
Comment (brianpetway) [reply src/js/components/glossary/Glossary.jsx]: remooooooved

**Review by nick-torres — APPROVED** [review]

**Review by nick-torres — APPROVED** [review]

**Pull request merged**